doggos/inference/takagi_sugeno_inference_system.py

```diff
@@ -1,6 +1,31 @@
+from typing import Dict
+
+from doggos.fuzzy_sets.membership import MembershipDegree
 from doggos.inference.inference_system import InferenceSystem
+from doggos.knowledge import Clause
```

Comment:
Change it to full path (doggos.knowledge.clause) import, it may cause problems otherwise.